tests/storage/test_state.py

```diff
@@ -369,7 +369,7 @@ def test_get_state_for_event(self):
         state_dict_ids = cache_entry.value
 
         self.assertEqual(cache_entry.full, False)
-        self.assertEqual(cache_entry.known_absent, {(e1.type, e1.state_key)})
+        self.assertEqual(cache_entry.known_absent, set())
```

Comment:
Note, this was a bit of a dodgy test, as `(e1.type, e1.state_key)` was not *known absent*, as it is in the returned dict.

Reply:
so the code was buggy before and is now fixed?

Reply:
Yeeees, though the way that it gets used means that the difference doesn't matter.